internals/daemon/api_exec_test.go

```diff
@@ -482,7 +482,7 @@ func (s *execSuite) TestExecChangeReady(c *C) {
 	rsp.ServeHTTP(rec, req)
 
 	c.Check(rec.Code, Equals, 500)
-	c.Check(rec.Body.String(), Matches, `.*cannot connect to websocket.*something went wrong.*`)
+	c.Check(rec.Body.String(), Matches, `.*cannot perform the following tasks.*something went wrong.*`)
```

Comment:
Maybe we should just shorten this to just `.*something went wrong.*` as we don't really care about the rest of the message, just that the error text we've set above is coming through.